schema.ts

```diff
@@ -568,6 +568,40 @@ export interface StringDateTimeSchema extends BaseStringSchema<ValueHintType.Dat
   timeFormat?: string;
 }
 
+/**
+ * State of outline on images that can be used with a {@link ImageSchema}.
+ */
+export enum Outline {
+  /** Image outline is disabled. */
+  Disabled = 'disabled',
+  /** Image outline is applied. */
+  Default = 'default',
```

Comment:
I don't think `Default` is a good name for this option. What if in the future we change the default in the product to disabled? Did developers intend to use the default regardless of what it was, or intend to turn outlines on?

What we do for other enums is have only explicit options and interpret no option set as "use the default". For example, the [`CurrencySchema`](https://github.com/coda/packs-sdk/blob/e9c08785a83d0073f82787a33d1a8d2f94910c78/schema.ts#L349) has a a `format` field that accepts an enum value. The [`CurrencyFormat`](https://github.com/coda/packs-sdk/blob/e9c08785a83d0073f82787a33d1a8d2f94910c78/schema.ts#L314) enum only has concrete options. If the developer doesn't specify which format to use, then they get the product default.

I suggest changing this to `Solid` or something that concretely defines this option.

Reply:
> This feels a bit oddly specific. I might call this `ImageCornerStyle` and have values of `Default` and `Rounded`, which leaves room for more styles in the future.

Based on this, would it seem reasonable to set something like `Standard` as without the styling for both properties, then `Rounded` and `Outlined` as the other enum values for now?